test/controllers/organizations/staff/activations_controller_test.rb

```diff
@@ -0,0 +1,34 @@
+require "test_helper"
+require "action_policy/test_helper"
+
+class Organizations::ActivationsControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @organization = ActsAsTenant.current_tenant
+    @staff = create(:admin)
+    sign_in @staff
+  end
+
+  context "authorization" do
+    include ActionPolicy::TestHelper
+
+    context "#update_activation" do
+      should "be authorized" do
+        assert_authorized_to(
+          :update_activation?, @staff, with: Organizations::ActivationsPolicy
+        ) do
+          patch activation_update_activation_url(@staff)
+        end
+      end
+    end
+  end
+
+  test "update activation should respond with turbo_stream when toggled on staff page" do
+    user = create(:super_admin)
+    sign_in user
+
+    patch activation_update_activation_url(@staff), as: :turbo_stream
```

Comment:
Hmm why not test the outcome rather than the mechanism? So, ideally we should compare the states of `@staff` before and then after to assert it changed as we expected. Does that make sense?